src/platform-includes/sourcemaps/upload/cli/javascript.mdx

```diff
@@ -1,20 +1,29 @@
-If you choose [`sentry-cli`](/product/cli/) as the method to upload your source files, we assume you already know how to generate source maps within your build tool chain.
+If you choose [`sentry-cli`](/product/cli/) as the method to upload your artifacts, we assume you already know how to generate source maps within your build toolchain.
 
-Sentry uses [releases](/product/releases/) to match the correct uploaded source maps to your events. You can use `sentry-cli` to create releases and upload source maps to Sentry.
+Sentry has two ways of matching the correct uploaded source maps to your events:
 
-## 1. Update SDK Options
+1. The first is to use Debug IDs, which allow Sentry to automatically associate minified sources with their respective source maps without needing to create a release. _This method is the preferred way of uploading artifacts._
 
-To be able to match Sentry errors with the correct release, you must provide a `release` property in the SDK options:
+2. The second is to use a [release](/product/releases/) to identify the artifacts you uploaded containing all your source files. _This method will require the creation of a release._
```

Comment:
So, you've articulated how these methods differ in uploading source maps, but I think it's better to focus on the implications of using releases here.

Reply:
Will fix it thanks!